user-guide/Advanced_Modules/Automation_module/Automation_script_actions/Email.md

```diff
@@ -19,6 +19,7 @@ Use this action to send a notification via email.
    > [!NOTE]
    >
    > - If you want to specify multiple indices for one table parameter, use a semicolon “;” as separator.
+   > - If you want to specify multiple parameters for one element, service, or protocol version, assign them all within a single line.
```

Comment:
```suggestion
   > - If you want to specify multiple parameters for one element, service, or protocol version, use a semicolon ";" as separator.
```

Reply:
It should not mention the ;. it should mention the original text.